io.github.smolblackcat.Progress.json

```diff
@@ -0,0 +1,152 @@
+{
+    "id": "io.github.smolblackcat.Progress",
+    "runtime": "org.gnome.Platform",
+    "sdk": "org.gnome.Sdk",
+    "runtime-version": "46",
+    "command": "progress-tracker",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "modules": [
+        {
+            "name": "gtkmm-4-10",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dmaintainer-mode=false"
```

Comment:
There should be two args that will reduce build time a bit:

```
"-Dbuild-demos=false",
"-Dbuild-tests=false"
```

Please check if it exists for this version.